RevenueCatUI/CustomerCenter/URLUtilities.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  URLUtilities.swift
+//
+//
+//  Created by Cesar de la Vega on 28/5/24.
+//
+
+import Foundation
+
+enum URLUtilities {
+
+    static func createMailURL() -> URL? {
+        let subject = "Support Request"
+        let body = "Please describe your issue or question."
+        let encodedSubject = subject.addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed) ?? ""
+        let encodedBody = body.addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed) ?? ""
+
+        // swiftlint:disable:next todo
+        // TODO: make configurable
+        let urlString = "mailto:support@revenuecat.com?subject=\(encodedSubject)&body=\(encodedBody)"
```

Comment:
We will be sending these from the backend soon. But we can address this todo in a separate PR 👍